tox.ini

```diff
@@ -42,6 +42,9 @@ commands =
 
 [testenv:lint]
 commands =
-    pylint {toxinidir}/tuf --ignore={toxinidir}/tuf/api
+    # Use different pylint configs for legacy and new (tuf/api) code
+    # NOTE: Contrary to what the pylint docs suggest, ignoring full paths does
+    # work, unfortunately each subdirectory has to be ignored explicitly.
+    pylint {toxinidir}/tuf --ignore={toxinidir}/tuf/api,{toxinidir}/tuf/api/serialization
```

Comment:
Is there a way to ignore all files under a certain folder?
That way you would be able to ignore everything under `tuf/api`.

Reply:
> Is there a way to ignore all files under a certain folder?

Nope (see my comment). I even looked at the implementation of  `--ignore` and `--ignore-patterns`.

Reply:
IIRC there's a corresponding ticket on the pylint issue tracker.